src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/View/ViewGenerator.cs

```diff
@@ -53,13 +53,12 @@ public class ViewGenerator : CommonGeneratorBase, ICodeGenerator
                 throw new ArgumentException(MessageStrings.ViewNameRequired);
             }
 
```

Comment:
I think it would make sense to put these first two checks into ValidateTemplate()